src/components/subscriptions/buttons/InviteLearnersButton.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+
+import InviteLearnersModal from '../../../containers/InviteLearnersModal';
+import ActionButtonWithModal from '../../ActionButtonWithModal';
+import { SubscriptionDetailContext } from '../SubscriptionDetailContextProvider';
+
+const InviteLearnersButton = ({ onSuccess, onClose }) => {
+  const { overview, subscription } = useContext(SubscriptionDetailContext);
+  return (
+    <ActionButtonWithModal
+      buttonLabel="Invite learners"
+      buttonClassName="add-users-btn"
```

Comment:
nit: along the lines of renaming AddUsers to InviteLearners, the `buttonClassName` could be `invite-learners-btn` instead.